email/registration.html

```diff
@@ -0,0 +1,26 @@
+<div>
+  <p>All Glories to Sri Sri Guru and Gauranga!</p>
+  <p>Hare Krsna. Dandavat pranams dear {{params.user.name}}.</p>
+  <br>
+
+  <p>Thank you for choosing to be a part of Srila Gurudeva’s Audio Seva Team!</p>
+  <p>We will soon email you further details about your seva assignment.</p>
+  <br>
+
+  <p>Your Servants,</p>
+  <p>Audio Seva Team.</p>
```

Comment:
Why not to take the common parts from the [sample](https://github.com/PureBhaktiArchive/audioseva/blob/release-1.0/email/sqr-allotment.html)?